crates/forge/src/pretty_printing.rs

```diff
@@ -5,11 +5,6 @@ use forge_runner::{test_case_summary::AnyTestCaseSummary, test_crate_summary::Te
 use starknet_api::block::BlockNumber;
 use std::collections::HashMap;
 
-pub fn print_warning(error: &Error) {
-    let warning_tag = style("WARNING").color256(11);
-    println!("[{warning_tag}] {error}");
-}
-
 pub fn print_error_message(error: &Error) {
```

Comment:
[nitpick]: Let's open an issue to move other print methods to shared create as well :+1: 
Depending on if we might want to use it i sncast, it should be assigned to either one of forge or chain interactions team.

Reply:
https://github.com/foundry-rs/starknet-foundry/issues/1740 - assigned to `chain-interactions` for now